src/shared/dlt_config_file_parser.c

```diff
@@ -494,8 +494,8 @@ int dlt_config_file_get_section_name(const DltConfigFile *file,
     if ((file == NULL) || (name == NULL) || (num < 0) || (num >= file->num_sections))
         return -1;
 
-    strncpy(name, (file->sections + num)->name, DLT_CONFIG_FILE_ENTRY_MAX_LEN);
-
+    strncpy(name, (file->sections + num)->name, DLT_CONFIG_FILE_ENTRY_MAX_LEN-1);
+    name[strlen((file->sections + num)->name)-1]='\0';
```

Comment:
Should be `name[DLT_CONFIG_FILE_ENTRY_MAX_LEN - 1] = '\0';`, I suppose. Please find the result in Travis CI: https://travis-ci.org/github/GENIVI/dlt-daemon/builds/723122895

Reply:
Agree, @ssugiura. Putting '\0' at the end is enough.